_config.yml

```diff
@@ -19,16 +19,21 @@ defaults:
     scope:
       path: ""
     values:
-      fullversion: "v1.9.0"
-      version: "v1.9"
+      fullversion: "v1.10.0"
```

Comment:
not sure if line 16 should be changed as well

Reply:
yes, it should be changed as well, if it was in Release 1.10